res/ai/DefaultAIPlayer/TaskStationMap.lua

```diff
@@ -496,16 +496,16 @@ function JobBuyStation:Tick()
 	if bestOffer ~= nil then
 		local price = bestOffer.GetTotalBuyPrice()
 		if bestOffer == bestAntennaOffer then
-			self:LogInfo("Buying antenna station in " .. bestOffer.GetSectionName(false) .. " at " .. bestOffer.pos.GetIntX() .. "," .. bestOffer.pos.GetIntY() .. ".  exclusive/increase: " .. bestOffer.GetExclusiveReach(false) .. "  price: " .. price)
-			TVT.of_buyAntennaStation(bestOffer.pos.GetIntX(), bestOffer.pos.GetIntY())
+			self:LogInfo("Buying antenna station in " .. bestOffer.GetSectionName(false) .. " at " .. bestOffer.x .. "," .. bestOffer.y .. ".  exclusive/increase: " .. bestOffer.GetExclusiveReach(false) .. "  price: " .. price)
+			TVT.of_buyAntennaStation(bestOffer.x, bestOffer.y)
```

Comment:
auch in Zeile 169/170 wird noch auf .pos zugegriffen

Reply:
Und beim Weiterspielen eines zuvor gespeicherten Spiels, in dem die KI schon mehrere Sender hat, kommt es zu Fehlerausgaben zu Zeile 398, da dort auf globale nil-Werte zugegriffen wird.

Reply:
Nein, vor der Umstellung treten diese Fehler nicht auf. Ich habe aber noch nicht analysiert, woher die Werte vorher kamen. Wie gesagt gibt es sowohl bei Laden eines alten Spielstands aber auch bei komplett neuem Spielstart nun Fehler beim KI-Senderkauf, die dazu führen, dass keine Sender gebaut werden. (Ich kann erst nach dem Wochenende weiteranalysieren)

Reply:
```

	-- 2) + 3)
	local requiredRandoms = math.max(minimumRequiredRandoms, maxToCheck - table.count(stationPositions))
	for i = 1, requiredRandoms do
		local x = math.random(35, 560)
		local y = math.random(1, 375)
		local newKey =  x .. "_" .. y
		stationPositions[newKey] = {x, y}
	end

	for i = 1, minimumRequiredRandomsWithPermission do
		local vec = TVT.of_GetRandomAntennaCoordinateInPlayerSections()
		if vec ~= nil then
			local newKey =  x .. "_" .. y
			stationPositions[newKey] = {x, y}
		end
	end
```
Zeile 398 ...

Reply:
x und y existieren nur, wenn die erste for loop durchlaufen wird...

Reply:
Letzter Commit sollte hier den Fix bereitstellen.